x/ibc-hooks/types/keys.go

```diff
@@ -0,0 +1,7 @@
+package types
+
+const (
+	ModuleName     = "ibchooks"
+	StoreKey       = "hooks-for-ibc" // not using the module name because of collisions with key "ibc"
```

Comment:
Can you explain this collision please? (just want to know for my own learning)

Reply:
Apparently StoreKeys can't share prefixes:  https://github.com/osmosis-labs/osmosis/pull/3589#discussion_r1044254058.

Reply:
My bad, didn't see that comment, thanks!